__tests__/extension-requests/extension-requests.test.js

```diff
@@ -586,4 +586,28 @@ describe('Tests the Extension Requests Screen', () => {
 
     expect(isTooltipVisible).toBe(true);
   });
+
+  test('Checks the Request Number and request value element on Extension requests listing page', async () => {
+    const url = 'http://localhost:8000/extension-requests/?dev=true'; // Include the dev parameter in the URL
+    await page.goto(url);
+
+    const extensionRequestNumberContainer = await page.$$(
+      '.extension-request-number',
+    );
+    let extensionRequestNumberText = extensionRequestNumberContainer[0];
+    expect(extensionRequestNumberText).toBeTruthy();
+
+    const cardNumber1Value = await extensionRequestNumberContainer[1].evaluate(
+      (node) => node.textContent,
+    );
+    expect(cardNumber1Value).toBe('5');
+
+    extensionRequestNumberText = extensionRequestNumberContainer[2];
+    expect(extensionRequestNumberText).toBeTruthy();
+
+    const cardNumber2Value = await extensionRequestNumberContainer[3].evaluate(
+      (node) => node.textContent,
+    );
+    expect(cardNumber2Value).toBe('1');
```

Comment:
- Each test should have single responsibility. 

- If this test fails a person without any context will just know that there is some issue with the extension request number but will not know the case for which it failed.

- To fix, please move this case(check for when there is no value from the api response) to a different test and update the test name to specify the exact case you are testing.

Reply:
Thanks for pointing that out! I have made the necessary changes and ensured each test has a single responsibility. Appreciate your input!